clarify code extension bullets (#7937)
fishtown-analytics/docs.getdbt.com: website/docs/docs/about-dbt-extension.md

# About the dbt VS Code Extension <Lifecycle status="preview" />

> The VS Code extension brings a hyper-fast, intelligent, and cost-efficient dbt development experience to VS Code.

Author: the www site also refers to it as vs code so updating this first bit. we might need to turn this into a constant variable so its consistent throughout the docs.

> The VS Code extension brings a hyper-fast, intelligent, and cost-efficient dbt development experience to VS Code.

Reviewer: It should include "dbt" since the name in the extension marketplace is that
```suggestion
The dbt VS Code extension brings a hyper-fast, intelligent, and cost-efficient dbt development experience to VS Code.
```

> This is the only way to enjoy all the power of the dbt Fusion engine while developing locally. 

Author: ```suggestion
This is the only way to enjoy all the power of the <Constant name="fusion_engine" /> while developing locally. 
```

- _Save time and resources_ with near-instant parsing, live error detection, powerful IntelliSense capabilities, and more.
- _Stay in flow_ with a seamless, end-to-end dbt development experience designed from scratch for local dbt development.

> The VS Code extension is available in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=dbtLabsInc.dbt). _Note, this is a public preview release. Behavior may change ahead of the broader generally available (GA) release._

Reviewer: Same as above
```suggestion
The dbt VS Code extension is available in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=dbtLabsInc.dbt). _Note, this is a public preview release. Behavior may change ahead of the broader generally available (GA) release._
```
